packages/mongodb-memory-server-core/src/util/__tests__/MongoInstance-test.ts

```diff
@@ -66,8 +62,6 @@ describe('MongodbInstance', () => {
       },
     });
     expect(inst.prepareCommandArgs()).toEqual([
-      '--bind_ip',
-      '127.0.0.1',
```

Comment:
How I remember there was some problem in old mongod versions that didn't start without IP binding. Anyway, let's try to launch it without an explicit IP address. Of course better to start on all interfaces. If we are wrong then somebody opens an issue ;)

Reply:
from my knowledge (what i read on the docs for mongod) is that mongod can start without any argument, and for bind_ip it will default to `localhost`

Reply:
Yep, I also read the same. But in practice, I had some problems, but I don't remember what exactly the issue was. And maybe it's already gone or I missed something a couple of years ago. Anyway let's keep your solution and if some problems occur we in the first place will look at this place.